### ISSUE TITLE
Getting error in flutter run

### ISSUE THREAD

**Issue by sanjeevinnobit** [bagisto/opensource-ecommerce-mobile-app#11]
**Getting error in Flutter run**

When we are running `flutter run` we are facing some errors.

These are the tasks that are failing:
1) Task :app:processDebugResources FAILED
2) Task :flutter_share:checkDebugAndroidTestAarMetadata FAILED
3) Task :just_audio:mergeDexDebugAndroidTest FAILED
AGPBI: {"kind":"error","text":"Cannot fit requested classes in a single dex file (# methods: 68452 > 65536)","sources":[{}],"tool":"D8"}
4) Task :video_player_android:mergeDexDebugAndroidTest FAILED
AGPBI: {"kind":"error","text":"Cannot fit requested classes in a single dex file (# methods: 67057 > 65536)","sources":[{}],"tool":"D8"}
5) AAPT2 process unexpectedly exit. 

**Expected behavior**
App should running.

**Tools that are using**
- Android Studio: Android Studio Koala (2024.1.1)
- Java (v17.0.10)
- Flutter version: 3.22.1
- Dart version: 3.4.1

**Comment by Faiyyaz** [comment]
I have fixed it for the latest flutter version

https://github.com/Faiyyaz/opensource-ecommerce-mobile-app

If you are okay with accepting PR then I will fork and raise a PR but for now sharing the link for my personal repository here

**Comment by bdsumon4u** [comment]
> I have fixed it for the latest flutter version
> 
> https://github.com/Faiyyaz/opensource-ecommerce-mobile-app
> 
> If you are okay with accepting PR then I will fork and raise a PR but for now sharing the link for my personal repository here

I'm sorry to say, you have disappointed me.
After failing so many times, I got hope seeing your link.
But it's 404. I'm disappointed.

**Comment by Faiyyaz** [comment]
@bdsumon4u Can you check now? Sorry it was a private repo I made it public

**Comment by bdsumon4u** [comment]
@Faiyyaz, yes. It's now public and thanks for that. But can you please tell me which version of flutter did you use?

**Comment by Faiyyazk** [comment]
The latest one as of that date I don't think in 2 weeks there were any big version release

**Comment by Faiyyaz** [comment]
The flutter release I used was latest one as of that date and I think there are no major new version after that.

**Comment by mebron** [comment]
Getting following errors:

/C:/Users/mebro/AppData/Local/Pub/Cache/hosted/pub.dev/carousel_slider-4.2.1/lib/carousel_slider.dart:9:1: Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
import 'carousel_controller.dart';
^^^^^^^^^^^^^^^^^^
lib/screens/product_screen/view/downloadable_product_options.dart:34:36: Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
  final buttonCarouselController = CarouselController();
                                   ^^^^^^^^^^^^^^^^^^
lib/screens/product_screen/view/downloadable_product_sample.dart:35:36: Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
  final buttonCarouselController = CarouselController();
                                   ^^^^^^^^^^^^^^^^^^
lib/screens/product_screen/view/product_image_view.dart:41:36: Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
  final buttonCarouselController = CarouselController();
                                   ^^^^^^^^^^^^^^^^^^
/C:/Users/mebro/AppData/Local/Pub/Cache/hosted/pub.dev/carousel_slider-4.2.1/lib/carousel_slider.dart:48:15: Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
            : CarouselController() as CarouselControllerImpl,
              ^^^^^^^^^^^^^^^^^^
/C:/Users/mebro/AppData/Local/Pub/Cache/hosted/pub.dev/carousel_slider-4.2.1/lib/carousel_slider.dart:62:15: Error: 'CarouselController' is imported from both 'package:carousel_slider/carousel_controller.dart' and 'package:flutter/src/material/carousel.dart'.
            : CarouselController() as CarouselControllerImpl,
              ^^^^^^^^^^^^^^^^^^
Target kernel_snapshot_program failed: Exception


FAILURE: Build failed with an exception.

* Where:
Script 'C:\flutter\packages\flutter_tools\gradle\src\main\groovy\flutter.groovy' line: 1687

* What went wrong:
Execution failed for task ':app:compileFlutterBuildDebug'.
> Process 'command 'C:\flutter\bin\flutter.bat'' finished with non-zero exit value 1

**Comment by Faiyyaz** [comment]
@mebron This is because the latest flutter has its own carousel slider can you downgrade from 3.24 and try again
